### DISCUSSION TITLE
Charger

### DISCUSSION THREAD

**Discussion by TimotejHofbauer** [PricelessToolkit/SOILSENS-V5W#4] [Ideas]
Hello,

project seems great but i was wondering why did you not include a charger? 
Do you have any recommendations on which charger can we use?  
Small 1s charger on the custom PCB with usbC port would be amazing, the idea is that I design a case which has enough room for the additional charger pcb (hardwired to your pcb). I am wondering also if there are any unwanted currents if the charger is connected constantly.

Something like that? Of course the current is too big. [aliexpres charger](https://www.aliexpress.com/item/1005005347930545.html?spm=a2g0o.detail.pcDetailTopMoreOtherSeller.4.3108Px1jPx1jkp&gps-id=pcDetailTopMoreOtherSeller&scm=1007.40050.354490.0&scm_id=1007.40050.354490.0&scm-url=1007.40050.354490.0&pvid=66a1edb1-818f-4909-889d-d27e55d45323&_t=gps-id:pcDetailTopMoreOtherSeller,scm-url:1007.40050.354490.0,pvid:66a1edb1-818f-4909-889d-d27e55d45323,tpp_buckets:668%232846%238116%232002&pdp_npi=4%40dis%21EUR%210.42%210.36%21%21%213.14%212.70%21%402103891017320159940072576ecc64%2112000042194010942%21rec%21SI%21169911332%21XZ&utparam-url=scene%3ApcDetailTopMoreOtherSeller%7Cquery_from%3A#nav-specification)

**Comment by PricelessToolkit** [comment]
Hi, USB connector and humid environment? I wouldn't say I like the idea, plus you need to charge the battery only once a year. So if you want you can use a module like that.
